src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -0,0 +1,1139 @@
+
+If you are not seeing an error that you expect to see in the **Errors** page, check the distributed tracing page since it does not exclude based on the `span.kind` attribute.
+
+
+## Errors inbox [#errors-inbox]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+
+## Metrics explorer [#metrics-explorer]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry:  🟡 Limited support
+
+New Relic APM agents use [timeslice metrics](/docs/data-apis/understand-data/metric-data/query-apm-metric-timeslice-data-nrql) while OpenTelemetry uses [dimensional metrics](/docs/data-apis/understand-data/new-relic-data-types/#dimensional-metrics).
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="trx-endpoint"
+    title="Metrics with transaction/endpoint dimensions"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌ 
+
+    The New Relic agent reports metrics (as well as spans and events) scoped to an individual transaction or endpoint.
+
+    Default OpenTelemetry SDK instrumentation does not consistently report such metrics. The semantic conventions for endpoint-level data, for example `http.target`, are still considered experimental.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="perf-spans"
+    title="Metrics on aggregate performance of spans (“transaction segments”)"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌ 
+
+    The New Relic agent measures the aggregate performance of spans within a request and reports them as metrics. We call these spans “segments” in the context of a transaction trace and record their “exclusive time” within the transaction, as well as the time spent in child spans/segments.
+
+    There is no notion of this in OpenTelemetry today.
+  </Collapser>
+</CollapserGroup>
+
+## Events explorer [#events-explorer]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅
+
+For OpenTelemetry services, you can use this page to explore data about the spans and logs emitted by your service. To explore data about your span events, click on **Span** and then **nr.spanEventCount**, or query for `SpanEvent`.
+
+
+## Explorer [#explorer]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry:  🟡 Limited support
+
+The APM view shows response time, throughput, error rate, end user response time, and PageView rates. The OpenTelemetry view only shows the first three metrics.
+
+## Lookout [#lookout]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry:  🟡 Limited support
+
+For OpenTelemetry services, only other OpenTelemetry-instrumented services are shown in this view.
+
+## Navigator [#navigator]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅
+
+## Apdex [#apdex]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌ 
+
+This concept is not covered by the OpenTelemetry specification.
+
+## Thread profiler [#thread-profiler]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌ 
+
+## VM support [#lang-specific]
```

Comment:
How about "Runtime/VM observability"? And change the anchor tag as appropriate.

Reply:
Done!